index.html

```diff
@@ -291,6 +292,109 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <footer class="footer">
+      <section id="contact" class="container contact">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form
+          action="https://formspree.io/f/mvodnloy"
+          method="post"
+          class="form"
+        >
+          <input
+            name="fullname"
+            type="text"
+            placeholder="John Doe"
```

Comment:
- [ ] Required: Kindly fix this and rename `John Doe` to `Full name` as per the contact form for mobile version in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
<img width="267" alt="Screenshot 2022-03-10 at 12 30 48 AM" src="https://user-images.githubusercontent.com/70269507/157539705-7d2e362d-357f-4e28-9e0c-a4fd125be3e8.png">